redis/content.md

```diff
@@ -1,44 +1,44 @@
 # What is Redis?
 
-Redis is an open-source, networked, in-memory, key-value data store with optional durability. It is written in ANSI C. The development of Redis is sponsored by Redis Labs today; before that, it was sponsored by Pivotal and VMware. According to the monthly ranking by DB-Engines.com, Redis is the most popular key-value store. The name Redis means REmote DIctionary Server.
+Redis is the world’s fastest data platform. It provides cloud and on-prem solutions for caching, vector search, and NoSQL databases that seamlessly fit into any tech stack&mdash;making it simple for digital customers to build, scale, and deploy the fast apps our world runs on.
 
-> [wikipedia.org/wiki/Redis](https://en.wikipedia.org/wiki/Redis)
+> [redis.io](https://redis.io)
```

Comment:
The original link was meant as the source for the paragraph above. I can't find this paragraph on `https://redis.io` so that doesn't seem to be the case now. That makes this new link a little odd in a quote block by itself.

Maybe the `redis.io` link should be incorporated into the first paragraph? Or expand the text slightly and drop the quote block?

```suggestion
Learn more at [redis.io](https://redis.io).
```

Reply:
We can address this in a follow up.